managed/services/checks/checks.go

```diff
@@ -189,6 +183,7 @@ func (s *Service) Run(ctx context.Context) {
 	defer s.l.Info("Done.")
 
 	s.CollectChecks(ctx)
+	s.initializeMetrics()
```

Comment:
I wonder do we need to initialize them at all? Set of labels become quite dynamic, it depends what was downloaded from Protal. So maybe it makes sense just to init them in place?

Reply:
On one hand yes - we can leave the init till the point when we actually need that label, on the other hand having no labels until checks are executed/failed is a bit weird too. So, I would keep it with the current logic: all labels are initialized at startup.

Reply:
I don't have strong opinion on that. Maybe @ritbl can suggest what is better for telemetry needs?

Reply:
These two solutions are effectively equivalent. In both cases we have initialization with `0` value.
This one looks a bit cleaner, though.

Reply:
@artemgavrilov, @0leksii and I discussed offline. It looks like we might not initialize all the possible metrics, hence it is better to avoid initialization to not confuse users and not produce wrong charts.
Thank you!